### PR TITLE
Add the ability to toggle the main bar component of `BarHitErrorMeter`

### PR DESCRIPTION
**Pull request by mk56-spn**
Title. 

Relevant discussion: https://github.com/ppy/osu/discussions/21669 

https://user-images.githubusercontent.com/74463310/208138897-2f931d08-e80f-490e-b30a-f12c90595ce7.mp4

should this have tests?

### CODE REVIEW

File: osu.Game/Screens/Play/HUD/HitErrorMeters/BarHitErrorMeter.cs [hunk 1]
```diff
@@ -42,6 +42,9 @@ public partial class BarHitErrorMeter : HitErrorMeter
         [SettingSource("Label style", "How to show early/late extremities")]
         public Bindable<LabelStyles> LabelStyle { get; } = new Bindable<LabelStyles>(LabelStyles.Icons);
 
+        [SettingSource("Bar visibility")]
+        public Bindable<bool> BarVisibility { get; } = new Bindable<bool>(true);
```
Comment (bdach) [inline osu.Game/Screens/Play/HUD/HitErrorMeters/BarHitErrorMeter.cs:46]: Is it just me or is this really non-descript? I'd probably go for "Show colour bars" or similar for the name of this setting.

Comment (mk56-spn) [reply osu.Game/Screens/Play/HUD/HitErrorMeters/BarHitErrorMeter.cs]: I didn't feel it to be as you say at first, but better safe than sorry so addressed.

**Comment by bdach** [comment]
Personally I'd make one more extremely subjective change and move the bool toggles together:

| before | after |
| :-: | :-: |
|  |  |

but I'm not super sure so I'll leave it. If more people agree then it could be changed here.

**Review by bdach — DISMISSED** [review]

**Comment by frenzibyte** [comment]
Agree on that. I actually had the exact same thought of moving it above the "show moving average arrow" toggle.

**Comment by mk56-spn** [comment]
had the same thought but opted for putting it at the bottom to preserve the order familiar to anyone who'd use it up till now, but this does look cleaner. eitherway i dont think its critical so : shrug :

**Review by peppy — APPROVED** [review]

**Pull request merged**